### ISSUE TITLE
Absolute paths for output folders stored in ui-config.json

### ISSUE THREAD

**Issue by AthanVigil** [AlUlkesh/stable-diffusion-webui-images-browser#194]
I recently had cause to move my automatic1111 folder from one drive to another, it appears that Image Browser stores the absolute path complete with drive letter into `ui-config.json`, i.e. 

`"image_browser/Images directorytxt2img/value": "D:\\work\\automatic1111\\stable-diffusion-webui\\outputs\\txt2img-images"`

It would be preferable to store the location as a relative path from the `stable-diffusion-webui` folder since that will work for moves within a drive or between drives.

automatic1111 version 1.3.1 with fully updated extensions to latest versions at time of writing.

**Comment by AthanVigil** [comment]
I just ran a test removing the `D:\\work\\automatic1111\\stable-diffusion-webui\\` prefix, the test confirms that a relative path stored in each of the path locations works as expected and will rectify the issue.

**Comment by AlUlkesh** [comment]
That wouldn't work in my case, since I've symlinked outputs to a HDD. Also I'm not sure how that would need to look like for Linux/Mac.

The browser already a functionality for cases like this:
![image](https://github.com/AlUlkesh/stable-diffusion-webui-images-browser/assets/99896447/6292819d-43b5-4d25-955c-95fa3f7e7b36)

**Comment by AthanVigil** [comment]
I saw the option to update the directory names in the database and assumed that would fix the issues. I updated all of the paths in the db to remove the prefix and and Image Browser still crashes when attempting to refresh the images. So I completely removed the Image Browsers extension folder.  Then re-started autmatic1111 and re-installed the Image Browser extension, no change. The entries in ui-config.json are sticky and still cause Image Browser to error.

I had assumed that if I removed the extension and re-installed it that it would then re-build any paths to the current locations. That appears to not be the case.

To demonstrate the issue, I had a fully working Image Browser this morning, I placed the prefix back into ui-conif.json: -

`"image_browser/Images directorytxt2img/value": "D:\\work\\automatic1111\\stable-diffusion-webui\\outputs\\txt2img-images"`

I restarted automatic1111 and tried to refresh the txt2img directory in Image Browser which produced the following error: -

```
2023-06-03-13:19:10 image_browser.py: get_image_page
2023-06-03-13:19:10 image_browser.py: get_all_images
2023-06-03-13:19:10 image_browser.py: curr_path: D:\work\automatic1111\stable-diffusion-webui\outputs\txt2img-images
Traceback (most recent call last):
  File "D:\automatic1111\stable-diffusion-webui\venv\lib\site-packages\gradio\routes.py", line 414, in run_predict
    output = await app.get_blocks().process_api(
  File "D:\automatic1111\stable-diffusion-webui\venv\lib\site-packages\gradio\blocks.py", line 1323, in process_api
    result = await self.call_function(
  File "D:\automatic1111\stable-diffusion-webui\venv\lib\site-packages\gradio\blocks.py", line 1051, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "D:\automatic1111\stable-diffusion-webui\venv\lib\site-packages\anyio\to_thread.py", line 33, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "D:\automatic1111\stable-diffusion-webui\venv\lib\site-packages\anyio\_backends\_asyncio.py", line 877, in run_sync_in_worker_thread
    return await future
  File "D:\automatic1111\stable-diffusion-webui\venv\lib\site-packages\anyio\_backends\_asyncio.py", line 807, in run
    result = context.run(func, *args)
  File "D:\automatic1111\stable-diffusion-webui\extensions\stable-diffusion-webui-images-browser\scripts\image_browser.py", line 928, in get_image_page
    filenames = get_all_images(img_path, sort_by, sort_order, keyword, tab_base_tag_box, img_path_depth, ranking_filter, ranking_filter_min, ranking_filter_max, aes_filter_min, aes_filter_max, score_type, exif_keyword, negative_prompt_search, use_regex, case_sensitive)
  File "D:\automatic1111\stable-diffusion-webui\extensions\stable-diffusion-webui-images-browser\scripts\image_browser.py", line 737, in get_all_images
    fileinfos = traverse_all_files(dir_name, [], tab_base_tag_box, img_path_depth)
  File "D:\automatic1111\stable-diffusion-webui\extensions\stable-diffusion-webui-images-browser\scripts\image_browser.py", line 442, in traverse_all_files
    f_list = [(os.path.join(curr_path, entry.name), entry.stat()) for entry in os.scandir(curr_path)]
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'D:\\work\\automatic1111\\stable-diffusion-webui\\outputs\\txt2img-images'
```
As you can see this induces the error even when the names in the database are correct for the current file locations. Hopefully the traceback is of some value.

The only way to prevent the crash was to manually remove the prefix from `ui-config.json`, I couldn't find anthing in the Image Browser settings page that would allow that. Given the varying skill levels of autmatic1111 users not everyone will be capable of finding the lines in `ui-config.json` to edit them and fix the issue.

Great job on the extension by the way it's a really useful addition to automatic1111.

**Comment by AlUlkesh** [comment]
Thanks. Yes, I assume when #195 is solved, this should also work as expected again.

**Comment by AlUlkesh** [comment]
I just merged the fix for #195. This should also fix this issue.

**Comment by AlUlkesh** [comment]
Great! Thanks.
